docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -5,55 +5,51 @@
 experimental[]
 
 {classification-cap} is a {ml} process for predicting the class or category of a 
-given data point in a dataset. Typical examples of {classification} problems are 
+given data point in a data set. Typical examples of {classification} problems are 
 predicting loan risk, classifying music, or detecting cancer in a DNA sequence. 
-In the first case, for example, our dataset consists of data on loan applicants 
+In the first case, for example, our data set consists of data on loan applicants 
 that covers investment history, employment status, debit status, and so on. 
 Based on historical data, the {classification} analysis predicts whether it is 
 safe or risky to lend money to a given loan applicant. In the second case, the 
 data we have represents songs and the analysis – based on the features of the 
 data points – classifies the songs as hip-hop, country, classical, or any 
 other genres available in the set of categories we have. Therefore, 
 {classification} is for predicting discrete, categorical values, unlike 
-{reganalysis} which predicts continuous, numerical values.
-
-From the perspective of the possible output, there are two types of 
-{classification}: binary and multi-class {classification}. In binary 
-{classification} the variable you want to predict has only two potential values. 
-The loan example above is a binary {classification} problem where the two 
-potential outputs are `safe` or `risky`. The music classification problem is an 
-example of multi-class {classification} where there are many different potential 
-outputs; one for every possible music genre. In the {version} version of the 
-{stack}, you can perform only binary {classanalysis}.
+{reganalysis} which predicts continuous, numerical values. The maximum number of 
+classes for {classanalysis} is 30.
 
 
 [discrete]
-[[dfa-classification-features]]
-==== {feature-vars-cap}
+[[dfa-classification-depvar]]
+==== {depvar-cap}
```

Comment:
> {depvar-cap}

This attribute doesn't exist in the shared attributes file, so it's not resolving.  I actually recommend omitting this section header and anchor.  I think this paragraph flows well from the one above it.